docs/github-connector/installation.md

```diff
@@ -0,0 +1,27 @@
+# GitHub Connector Installation
+
+- [GitHub Connector Installation](#github-connector-installation)
+  - [Installation in Kyma with Helm](#installation-in-kyma-with-helm)
+    - [Prerequisites](#prerequisites)
+    - [Steps](#steps)
+
+## Installation in Kyma with Helm
+
+### Prerequisites
+
+- Connection to Kyma cluster
+- GitHub Connector Docker image
+
+### Steps
+
+1. Go to [Kyma repository](https://github.com/kyma-project/kyma) and run script `/installation/scripts/tiller-tls.sh` to get certificates needed for using Helm commands. By default they are stored in `~/.helm` directory. After that add `--tls` flag to every Helm command to authorize and authenticate yourself
+2. Go to `chart/githubconnector` directory. Run the command to install GitHub Connector:
+
+    ``` shell
+    helm install --set container.image={DOCKER_IMAGE} --set kymaAddress={KYMA_ADDRESS} -n {RELEASE_NAME} . --tls
+    ```
+
+    >**CAUTION:** Kyma address should be in the right format. It must consist of domain name, without dot  character at the beggining, for example `35.187.32.214.xip.io`
```

Comment:
```suggestion
    >**CAUTION:** Make sure that the Kyma address is in the correct format. It must consist of the domain name, without the dot character at the beginning. For example, `35.187.32.214.xip.io`.
```
`It must` or `it consists`?
- Don't use `should`. See [this](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#terminology).
- Don't use `for example` in the middle of the sentence. See [this](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#terminology).
- I'd rephrase that:
  > It must consist of the domain name, **without the dot character at the beginning**.

  to something like:
  >  It consists of the domain name and omits the dot at the beginning. 

  if the dot is there in other cases and just here needs to be omitted.

Reply:
Actually, the dot doesn't appear there usually and most people wouldn't put it there, but I wanted to emphasize it. I'll change it to `It consists of the domain name and cannot begin with the dot.`